Formula/s/sqlpage.rb

```diff
@@ -1,10 +1,10 @@
 class Sqlpage < Formula
-  desc "Web application framework, for creation of websites with simple database queries"
-  homepage "https://sql.ophir.dev/"
-  url "https://github.com/lovasoa/SQLpage/archive/refs/tags/v0.31.0.tar.gz"
+  desc "SQL-only web application builder, to quickly create dynamic webapps with simple database queries"
```

Comment:
the line is too long
```suggestion
  desc "Web app builder using SQL queries to create dynamic webapps quickly"
```

Reply:
Thanks ! I committed your suggestion.